### PR TITLE
Feature/detail

### PR DESCRIPTION
**Pull request by beto-toro**

### CODE REVIEW

File: src/components/Navbar/index.js [hunk 1]
```diff
@@ -5,30 +5,40 @@ import Search from '../Search';
 
 import Logo from '../../assets/images/logoQuarto.svg';
 import HamburguerMenu from '../../assets/images/menu-hamburguer.svg';
+import HamburguerMenuBlack from '../../assets/images/menu-hamburguer-black.svg';
 import './styles.scss';
 
-const Navbar = () => {
+const Navbar = (color__change) => {
+  const { prop } = color__change;
```
Comment (MigueMartelo) [inline src/components/Navbar/index.js:12]: Here the best way is that the name be `color_change` instead `prop`, is better when we have a name more descriptive than a generic name.

File: src/pages/Detail/index.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+          <div className="photos__main"></div>
+          <div className="photos__container">
+            <div className="carrousel">
+              <div className="caja"></div>
+              <div className="caja"></div>
+              <div className="caja"></div>
+              <div className="caja"></div>
+              <div className="caja"></div>
+              <div className="caja"></div>
+              <div className="caja"></div>
```
Comment (MigueMartelo) [inline src/pages/Detail/index.js:29]: Remember that the list of photos must be a `ul` element 😉

File: src/components/Navbar/index.js (hunk 1 above)
Comment (beto-toro) [reply src/components/Navbar/index.js]: done 💪🏼

File: src/pages/Detail/index.js (hunk 2 above)
Comment (beto-toro) [reply src/pages/Detail/index.js]: done 💪🏼

**Pull request merged**